ST_mortality.c

```diff
@@ -588,29 +652,39 @@ static void _succulents( const SppIndex sp) {
 }
 
 
-/***********************************************************/
+/**
+ * \brief Implements slow growth mortality.
+ * 
+ * Kill plants based on a probability if the growth rate
+ * is less than the "slow rate" which is defined by the
+ * user in the group-level parameters (max_slow) and in
+ * the species-level parameters (max_rate). The slow rate
+ * is growthrate <= max_slow * max_rate.
+ *
+ * Increment the counter for number of years of slow growth.
+ * If the number of years of slow growth is greater than
+ * max_slow (defined in species.in), draw a random number
+ * and test it against the probability of mortality.  C&L'90
```

Comment:
replace "test it against" with "compare it to"

Reply:
Addressed by commit 984bd13.